machine-learning/ml/preprocessing/preprocessing.py

```diff
@@ -50,6 +49,9 @@ def retrieve_labelled_instances(dataset, refactoring: LowLevelRefactoring):
     log("refactoring instance (after dropping NA)s: {}".format(refactored_instances.shape[0]))
     log("not refactoring instances (after dropping NA)s: {}".format(non_refactored_instances.shape[0]))
 
+    # Remove columns from refactored instances that are not in stable commits
+    refactored_instances = refactored_instances.drop(['refactoring', 'refactoringSummary'], axis=1)
```

Comment:
In the past, those columns were removed in the SQL queries already. Did it change?